eks/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    bucket = "constellationlabs-dev-tf"
```

Comment:
Is this the same bucket that is used across all our repos? It should be the same but with different key (just eks, without env suffix), and then dev/prod workspace with proper access policy, so the prod env can be deployed on prod AWS account only and others on dev

Reply:
It's a new bucket. So you suggest using the same bucket for prod and for dev? If we would go this path, then as a user that's deploying something to the dev environment, you would need to have access to the prod bucket. It doesn't seem to be correct.

>with proper access policy, so the prod env can be deployed on prod AWS account only and others on dev

What kind of access policy is that?

Reply:
No, you don't need an access to prod account. All but prod envy would allow you to deploy on dev AWS only, with restricted access to prod. You won't be able to execute terraform apply on prod workspace. Admin needs to add the policy to existing tf bucket, let's talk on slack about that.

Reply:
Thanks for the explanation. I understand the reasoning behind it. Fixing it.